### DISCUSSION TITLE
How can I use oxc as one of the husky(commit-lint) hooks?

### DISCUSSION THREAD

**Discussion by adoin** [oxc-project/oxc#1670] [Q&A] [answered]
It seems for now can not auto formatter, what if only check "ye or no" before commit?

**Comment by Boshen** [comment, marked as answer]
It's easiest to create a `lint-staged.config.mjs`:

```javascript
export default {
  "**/*.{ts,js,mjs}": () => "oxlint ."
}
```

This will lint the whole project.

**Reply by adoin** [reply]
Will it be long time to see formatter?

**Reply by Boshen** [reply]
We'll pick up progress after the holiday seasons. You are welcomed to contribute 😀
